lib/rex/proto/mssql/client.rb

```diff
@@ -27,6 +27,7 @@ class Client
         attr_accessor :ssl_cipher
         attr_accessor :proxies
         attr_accessor :connection_timeout
+        attr_accessor :realm
```

Comment:
Any context on this change? 👀

Reply:
Forgot to rip it out, was using this (incorrectly) for cwd